### ISSUE TITLE
Detect missing Conditions

### ISSUE THREAD

**Issue by pda** [awslabs/cfn-python-lint#92]
When a resource specifies a `Condition: …` and the named condition does not exist, `cfn-lint` does not detect a problem.

```yaml
  ListenerHttps:
    Type: AWS::ElasticLoadBalancingV2::Listener
    Condition: ListenForHttps
    …
```

CloudFormation server-side validation says:

> An error occurred (ValidationError) when calling the CreateStack operation: Template format error: Condition ListenForHttps is not defined.

---

(Apologies for Issue rather than Pull Request, if I have time I'll come back with code)

**Comment by fatbasstard** [comment]
Has been fixed already (e.g. https://github.com/awslabs/cfn-python-lint/pull/48)

Will be available with a new release (https://github.com/awslabs/cfn-python-lint/pull/84)

**Comment by pda** [comment]
Excellent, thanks.
